ArcadiaParties-SPA/src/app/user/services/user.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { User } from '../models/User';
+
+@Injectable({
+  providedIn: 'root'
+})
+export class UserService {
+  public getUser() {
```

Comment:
This is arguable, because some developers prefer to have explicit visibility modifier. So, it depends on code-style, but we haven't defined any rules about this.